salt/metalk8s/salt/master/configured.sls

```diff
@@ -9,6 +17,8 @@ Configure salt master:
     - mode: '0644'
     - makedirs: true
     - backup: false
+    - defaults:
```

Comment:
I think you need to add `template: jinja` here

Reply:
Indeed, and then rename `master_99-metalk8s.conf` to `master_99-metalk8s.conf.j2`

Reply:
Thanks guys :+1: